docker-compose.test.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.9'
```

Comment:
So we can use `profiles` which prevents the `playwright` container from starting on `up`